lib/grpcServer/handlers/platform/broadcastStateTransitionHandlerFactory.js

```diff
@@ -52,12 +51,15 @@ function broadcastStateTransitionHandlerFactory(rpcClient, handleAbciResponseErr
       throw error;
     }
 
-    if (result.code !== undefined && result.code !== 0) {
-      const { error: abciError } = JSON.parse(result.log);
+    if (result.code === undefined) {
```

Comment:
I guess we don’t want to duplicate this code everywhere. Why not just move it to createGrpcError... function?

Reply:
Then we need to pass result to createGrpcError, right?